### PR TITLE
add modal warning for no account, change loader

### PR DESCRIPTION
**Pull request by mmvsha**

### CODE REVIEW

File: src/components/MarketProgress/index.js [hunk 1]
```diff
@@ -14,11 +14,12 @@ const MarketProgress = ({
...
-  
-  if (failed) {
-    strokeDasharray = 0
-  }
+  // its not used
+  // let strokeDasharray = Math.abs((progress) - 1) * Math.PI * (190 * 2)
+
+  // if (failed) {
+  //   strokeDasharray = 0
+  // }
```
Comment (andre-meyer) [inline src/components/MarketProgress/index.js]: Feel free to remove it, this is from `ProgressSpinner`, I forgot to remove it

File: src/containers/App/index.js [hunk 2]
```diff
@@ -31,20 +37,37 @@ class App extends Component {
...
+          </CSSTransition>
+        </TransitionGroup>
+        <Modal
+          isOpen={this.props.blockchainConnection && !this.props.account}
+          contentLabel="no-account-modal"
+          style={modalStyles}
+        >
+          <h1 id="heading">Oops!</h1>
+          <div id="description">
+            <p>We couldn't detect your account. Please check your wallet provider and reload the page</p>
```
Comment (andre-meyer) [inline src/containers/App/index.js]: Can we fix this problem in Github where the syntax always messes up? That's btw why I always escape `'`

File: src/components/Icons/loading.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        from="0 20 20"
+        to="360 20 20"
+        dur="0.5s"
+        repeatCount="indefinite"
+      />
+    </path>
+  </svg>
+)
+
+export default LoadingIcon
```
Comment (andre-meyer) [inline src/components/Icons/loading.js]: Can we use the `ProgressSpinner` instead maybe? Or is this completely different? I want to keep the loading indicators similar

Comment (andre-meyer) [reply src/components/Icons/loading.js]: Ah, the Loading Icon is more for undetermined length. Maybe you can add that to the ProgressSpinner, or create a wrapper that handles it.

Comment (mmvsha) [reply src/components/Icons/loading.js]: @andre-meyer Should I? `ProgressSpinner` is a component for showing progress, and this is just a wrapper on svg icon

Comment (andre-meyer) [reply src/components/Icons/loading.js]: But it's a loading indicator kind of thing right? Maybe you can reuse some of it so it looks similar?

**Pull request merged**